src/test/kotlin/no/nav/syfo/behandlendeenhet/api/internad/BehandlendeEnhetApiSpek.kt

```diff
@@ -193,187 +160,14 @@ class BehandlendeEnhetApiSpek : Spek({
-                        setBody(behandlendeEnhetUpdateDTO)
-                    }
-                    response.status shouldBeEqualTo HttpStatusCode.OK
-
-                    val oppfolgingsenhet = repository.getOppfolgingsenhetByPersonident(PersonIdentNumber(behandlendeEnhetDTO.personident))
-                    oppfolgingsenhet?.enhetId shouldBe null
-                    oppfolgingsenhet?.personident?.value shouldBeEqualTo ARBEIDSTAKER_PERSONIDENT.value
-
-                    val kafkaRecordSlot = slot<ProducerRecord<String, KBehandlendeEnhetUpdate>>()
-                    verify(exactly = 1) { kafkaProducerMock.send(capture(kafkaRecordSlot)) }
-                    kafkaRecordSlot.captured.value().personident shouldBeEqualTo oppfolgingsenhet?.personident?.value
-                    kafkaRecordSlot.captured.value().updatedAt shouldBeEqualTo oppfolgingsenhet?.createdAt
-                }
-            }
-
-            it("should update oppfolgingsenhet if already in db") {
-                testApplication {
-                    val client = setupApiAndClient()
-                    val response = client.post(personUrl) {
-                        bearerAuth(validToken)
-                        header(HttpHeaders.ContentType, ContentType.Application.Json.toString())
-                        setBody(behandlendeEnhetDTO)
-                    }
-                    response.status shouldBeEqualTo HttpStatusCode.OK
-                    val oppfolgingsenhet = repository.getOppfolgingsenhetByPersonident(PersonIdentNumber(behandlendeEnhetDTO.personident))
-                    oppfolgingsenhet?.enhetId?.isNavUtland() shouldBeEqualTo true
-
-                    val updatePersonDTO = behandlendeEnhetDTO.copy(isNavUtland = false, oppfolgingsenhet = null)
-                    val responsePost = client.post(personUrl) {
-                        bearerAuth(validToken)
-                        header(HttpHeaders.ContentType, ContentType.Application.Json.toString())
-                        setBody(updatePersonDTO)
-                    }
-                    responsePost.status shouldBeEqualTo HttpStatusCode.OK
-
-                    val oppfolgingsenhetUpdate = repository.getOppfolgingsenhetByPersonident(PersonIdentNumber(updatePersonDTO.personident))
-                    oppfolgingsenhetUpdate?.enhetId shouldBe null
-
-                    val kafkaRecordSlot = mutableListOf<ProducerRecord<String, KBehandlendeEnhetUpdate>>()
-                    verify(exactly = 2) { kafkaProducerMock.send(capture(kafkaRecordSlot)) }
-                    kafkaRecordSlot[0].value().personident shouldBeEqualTo oppfolgingsenhetUpdate?.personident?.value
-                    kafkaRecordSlot[1].value().updatedAt shouldBeEqualTo oppfolgingsenhetUpdate?.createdAt
-                    kafkaRecordSlot[0].value().updatedAt shouldBeLessThan kafkaRecordSlot[1].value().updatedAt
-                }
-            }
-        }
-
-        describe("Unhappy path") {
```

Comment:
Jeg slettet happy-path testene for endepunktet da dette var dekket av testene for det nye endepunktet. Kopierte derimot de relevante unhappy-testene og limte de inn lenger ned.